src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -3,227 +3,65 @@ tags: runtime-mobileandreactiveweb;
 summary: 
```

Comment:
@annettewilliamsoutsystems We're missing summary sentences for the reviewed patterns (also in the new reference topics).